Desktop/utilities/qutils.cpp

```diff
@@ -241,3 +244,83 @@ QString shortenWinPaths(QString in)
 	return in;
 #endif
 }
+
+
+//Turning QMLENGINE_DOES_ALL_THE_WORK on also works fine, but has slightly less transparent errormsgs so isn't recommended
+//#define QMLENGINE_DOES_ALL_THE_WORK
```

Comment:
I guess we want this as an option, if so, I think it can probably go into the `JASP.cmake`.

Reply:
Yeah no not really, that was just something I did for debugging but it turned out the problem was somewhere else entirely.
And the code could turn out to be useful so I hid it under that.